README.md

```diff
@@ -39,10 +39,12 @@ You can also download the Mobile SDKs from S3 following these steps:
 * Unzip, copy both `amazon-chime-sdk.aar` and `amazon-chime-sdk-media.aar` into the `android/app/libs` folder
 
 ### iOS
-* Download the Amazon Chime SDK binaries
-    - [AmazonChimeSDK-0.18.0.tar.gz](https://amazon-chime-sdk-ios.s3.amazonaws.com/sdk/0.18.0/AmazonChimeSDK-0.18.0.tar.gz)
-    - [AmazonChimeSDKMedia-0.14.0.tar.gz](https://amazon-chime-sdk-ios.s3.amazonaws.com/media/0.14.0/AmazonChimeSDKMedia-0.14.0.tar.gz)
-* Unzip, copy both `AmazonChimeSDK.framework` and `AmazonChimeSDKMedia.framework` into the `ios/RNDemo` folder
+#### From CocoaPods
+With `pod 'AmazonChimeSDK-Bitcode', '0.19.1'` declared in the `ios/Podfile`, the specified version of `AmazonChimeSDK-Bitcode` pod will be downloaded and integrated when you run `pod install` in the step 5 below.
```

Comment:
For Android, we still keep "Download from S3" section, should we also keep it for iOS?

Reply:
I think the intent is to use package management tools only in RN demo.